services/web/src/utils/analytics/README.md

```diff
@@ -0,0 +1,180 @@
+# GTM Tracking
```

Comment:
Let's be clear here: Google Tag Manager / Google Analytics

Reply:
Updated